Geometry/HcalSimData/data/CaloUtil/2026/v0/CaloUtil.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0"?>
```

Comment:
What does "v0" mean? We usually start with "v1".

Reply:
OK I shall do that